packages/quantic/force-app/main/default/lwc/quanticResult/quanticResult.js

```diff
@@ -29,12 +29,16 @@ export default class QuanticResult extends LightningElement {
       return undefined;
     }
     switch (objType.toLowerCase()) {
```

Comment:
Instead of a switch, what would you think of using a map?

    get objectTypeIcon() {
        const objType = this.result.raw.objecttype;
        if (!objType) {
            return undefined;
        }

        const defaultIcon = `standard:${objType.toLowerCase()}`;
        const iconMap = {
            feeditem: 'standard:feed',
            'how_to': 'standard:question_feed',
            ...
        };

        const icon = object.keys(iconMap).find((key) => key === objType.toLowerCase());
        return icon ?? defaultIcon;
    }